README.md

```
@@ -134,6 +138,7 @@
 | [catsmile](https://www.catsmile.cloud/)  | <https://grpc-axone.catsmile.space> |
 | [Winsnip](https://service.winsnip.xyz) | <https://grpc-axone.winsnip.site> |
 | [NodeVault](https://nodesvault.com) | <https://grpc.nodesvault.com> |
+| [Vinjan.Inc](https://service.vinjan.xyz) | <https://grpc-axone.vinjan.xyz> | ✅ |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)